controller/main/main.ino

```diff
@@ -1,11 +1,30 @@
 /*
  * QRGX Microcontroller for 2018 SEDS. This is the main file.
  * 
- * Author(s): Viraj Bangari
+ * Author(s): Viraj Bangari, Aaron Rosenstein
  * 
  * Date: May 2018
  */
 
+//Digital Pin Declarations:
+const int actuator_PWM[]={2,8};
+const int actuator_2[]={3,7};
+const int actuator_1[]={4,6};
+const int actuator_STBY=5;
+
+//Analog Pin Declarations:
+const int potentiometer[]={0,1}; //Actuators
+const int therm_1=2; //Heating Block Thermistors
+const int therm_2=3;
+const int therm_3=4;
+const int therm_4=5; //Dry Block Heater Thermistor
+const int amb_therm_1=6; //Ambient Temp Sensors
+const int amb_therm_2=7; 
+
+//Variable Declarations:
+int minActuatorValue[2];
```

Comment:
Minor thing - C standard doesn't specify what the default initialized values in an Array are. For instance, if you try to call `minActuatorValue[0]` without setting it first, you can potentially get some really large random number. Or a really small random number. Or zero. We don't know since it is not guaranteed.

The safer option would be to initialize it to same safer, known value (maybe 0 in this case), similar to how you digit for potentiometer.